traffic_ops/traffic_ops_golang/hwinfo/hwinfo.go

```diff
@@ -20,74 +20,116 @@ package hwinfo
  */
 
 import (
-	"errors"
+	"encoding/json"
+	"fmt"
 	"net/http"
+	"strconv"
 
 	"github.com/apache/trafficcontrol/lib/go-log"
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/apache/trafficcontrol/lib/go-util"
+
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/dbhelpers"
+
 	"github.com/jmoiron/sqlx"
 )
 
+const selectHWInfoQuery = `
+SELECT
+	s.host_name as serverhostname,
+	h.id,
+	h.serverid,
+	h.description,
+	h.val,
+	h.last_updated
+FROM hwInfo h
```

Comment:
the table is `hwinfo`, but I guess the table names are case-insensitive?

Reply:
They are in queries, unless quoted. But better to use the real name anyway, safer and less confusing.

Reply:
Ah, that was probably tab-completion